server/types.graphql

```diff
@@ -16,7 +16,7 @@
    facebookUserId: String @isUnique
    password: String
    phoneNum: String
-   email: String # optional, because it's obtained from Facebook API
+   email: String @isUnique # optional, because it's obtained from Facebook API
```

Comment:
The comment here might have to be deprecated. Not sure it provides a solid explanation. Better still, if it can be restructured, it's fine.

Reply:
Yip.